tests/Integration/BoxErrorTest.php

```diff
@@ -58,11 +58,12 @@ public function testExceptionWithErrorIsThrown() : void
             self::assertSame('AccessDeniedError', $error->getType());
             self::assertSame("Write access to space '_priv' is denied for user 'user_with_no_privileges'", $error->getMessage());
             self::assertSame(42, $error->getCode());
-            self::assertEquals([
+            /* Checks that error fields match expected. */
```

Comment:
Could you also update [the comment above](https://github.com/tarantool-php/client/blob/b3e9287bd2338bb3c4a8b2ada6040d2815d9c5f9/tests/Integration/BoxErrorTest.php#L36-L40) by mentioning the new field?

Reply:
Done.

```diff
--- a/tests/Integration/BoxErrorTest.php
+++ b/tests/Integration/BoxErrorTest.php
@@ -35,7 +35,7 @@ final class BoxErrorTest extends TestCase
         try {
             /*
              * Triggers "AccessDeniedError", which includes the fields
-             * "object_type", "object_name" and "access_type".
+             * "object_type", "object_name", "access_type", "user" (since 3.1).
              * See https://www.tarantool.io/en/doc/2.4/dev_guide/internals/box_proto>
              */
             $this->client->evaluate("
```